tests/cypress/integration/marketplace.spec.js

```diff
@@ -1,60 +1,130 @@
 // <reference types="Cypress" />
+const marketplace = require('../fixtures/products.json');
 
 describe('Marketplace Page', function () {
 
 	before(() => {
+		cy.server();
+		cy.route('GET', '**/newfold-marketplace/v1/marketplace**', 'fx:marketplace').as('marketplace');
```

Comment:
@circlecube We should probably be using `cy.intercept()` here instead of `cy.route()`. This could have something to do with why you were having trouble with the Fetch API. See https://docs.cypress.io/api/commands/intercept#Comparison-to-cy-route